rct229/rulesets/ashrae9012019/ruleset_functions/get_swh_uses_associated_with_each_building_segment.py

```diff
@@ -0,0 +1,28 @@
+from rct229.utils.jsonpath_utils import find_all
+
+
+def get_swh_uses_associated_with_each_building_segment(
+    rmd: dict, building_segment_id: str
```

Comment:
Hi Jiarong, For this function, it seems like we don't need the ```building_segment``` argument according to RDS. Can you double-check this?

Reply:
> Hi Jiarong, For this function, it seems like we don't need the `building_segment` argument according to RDS. Can you double-check this?

Thanks Yun Joon. I realized it is not used so I removed it in the latest commit.

Reply:
Sorry, I was confusing myself. I thought you developed ```get_swh_uses_associated_with_each_building_segment``` this function in this PR, but I realized that you pulled its branch to here. Sorry for the confusion. I believe your original development is correct.